### ISSUE TITLE
Don't allow to leave page when you edit text in input box

### ISSUE THREAD

**Issue by utzer** [friendica/friendica#2371]
Can we have a function that prevents to leave/close a page if editing text on that page? This would be nice to prevent accidental closing a tab when entering text for an reply. This might be used for certain fields only.

**Comment by tobiasd** [comment]
I would say, a warning at most, but not total prevention. For instance it happens that I write comments full of anger about the posting above and then leave the page on purpose not sending the reply but could "work on my anger" to calm down a bit.

**Comment by ben-utzer** [comment]
Yes, I mean a warning, if you want to leave it ask "Do you really want to discard and leave the page?" With yes/no/abort or something similar.
